src/pages/friend.js

```diff
@@ -54,13 +50,14 @@ function Friend() {
                   Vacation week!
                 </div>
 
-                <div className="w-7/12 mt-6 text-base text-white">
+                <div className="mt-6 text-base text-white w-full">
                   Well, there ain't one week like the other, right?
                   <br />
                   <br /> This time you haven't got anyone, no worries. Maybe,
-                  it's just a sign from the Universe to have some me-time, get a
-                  nice drink and devote 15 min to yourself. Don't forget to
-                  answer the question of the week and get back next week!{" "}
+                  it's just a sign from the <br /> Universe to have some
+                  me-time, get a nice drink and devote 15 min to <br />
+                  yourself. Don't forget to answer the question of the week and
+                  get back next week!
```

Comment:
Using `<br/>` is tricky. You should prioritize the use of `<p>` paragraphs instead and manage the style with classes.